src/InputMapper/ArrayInputMapper.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-cli for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-cli/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-cli/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Laminas\Cli\InputMapper;
+
+use Symfony\Component\Console\Input\InputInterface;
+
+use function ltrim;
+use function strpos;
+
+final class ArrayInputMapper implements InputMapperInterface
+{
+    /** @var string[] */
+    private $map;
+
+    public function __construct(array $map)
+    {
+        $this->map = $map;
+    }
+
+    public function __invoke(InputInterface $input) : array
+    {
+        $params = [];
+        foreach ($this->map as $old => $new) {
+            $params[$new] = strpos($old, '-') === 0
+                ? $input->getOption(ltrim($old, '-'))
```

Comment:
I'm a little confused by this logic. Are the maps supposed to include the `--` or `-` prefixes? I would think we'd want them to be just the option _names_.

Reply:
This is the same approach we have in symfony console, arguments are without `--` and options are with `--`, it's gonna probably also work for shortcut options (with single `-`).

See example in: https://symfony.com/doc/current/console/calling_commands.html